virttest/utils_libvirt/libvirt_bios.py

```diff
@@ -29,3 +29,27 @@ def remove_bootconfig_items_from_vmos(osxml):
     if os_attrs.get("firmware"):
         osxml.del_firmware()
     return osxml
+
+
+def check_boot_config(session, test, check_list=None):
+    """
+    Check /boot/config-$KVER file.
+
+    :param session: vm session.
+    :param test: test object.
+    :param check_list: checking list, default None.
+    :raises: test.fail if checking fails.
+    """
+    if not check_list:
+        check_list = [
```

Comment:
This specific checking list are not suitable in the common utility function as default values. They are better to be in the tp-libvirt case.

Reply:
Thanks, Updated